couchbase-sdk-cxx-black-duck-manifest.yaml

```diff
@@ -2,13 +2,13 @@
 components:
   boringssl:
     bd-id: 1e303b15-b82b-4a4d-8cd6-a2048fb2c73d
-    versions: [ 20200921 ]
+    versions: [ 20240201 ]
```

Comment:
Here I'm not sure. The old one `20200921` did not reflect the actual version being used, but rather some rather recent version existing in BlackDuck database. Maybe we should rather ask them to add this one. It is being used in current grpc repo:

https://github.com/grpc/grpc/blob/db51a3f69ad6de2d4cce0f7642b829e0ebf6909e/bazel/grpc_deps.bzl#L246-L247

Reply:
Oh -- sorry!  I didn't see this comment.  I think we either need to follow-up w/ the build team (re: CBD-5677) or maybe we can merge the changes and see how the BD scan goes?  We can just be sure to update the yaml prior to the next release.

Reply:
I have extracted BoringSSL update into separate PR: https://github.com/couchbaselabs/couchbase-cxx-client/pull/544